src/controllers/filters/FilterSet.hpp

```diff
@@ -1,86 +1,30 @@
 #pragma once
 
```

Comment:
Let's include the very clearly used things here like QList and QMap to ensure we don't easily break things in the future